gsa/src/web/entity/__tests__/note.js

```diff
@@ -22,6 +22,7 @@ import {setLocale} from 'gmp/models/date';
 import Note from 'gmp/models/note';
 
 import {rendererWith} from 'web/utils/testing';
+import {setTimezone} from 'web/store/usersettings/actions';
```

Comment:
```suggestion
import {setTimezone} from 'web/store/usersettings/actions';
import {rendererWith} from 'web/utils/testing';
```

Reply:
Can you explain the ordering of the imports to me @sarahd93 ?

Reply:
Our coding guide for gsa says "alphabetically within the above categories" so I figured web/store comes before web/utils.